toolchains/lowrisc_toolchain_rv32imc/lowrisc_toolchain_rv32imc.bzl

```diff
@@ -79,41 +79,49 @@ def _get_additional_system_include_paths(ctx):
 
 def _lowrisc_toolchain_rv32imc_toolchain_config_info_impl(ctx):
     tool_paths = [
+        tool_path(
+            name = "cpp",
+            path = "clang_wrappers/{os}/cpp",
+        ),
         tool_path(
             name = "gcc",
-            path = "gcc_wrappers/{os}/gcc",
+            path = "clang_wrappers/{os}/gcc",
+        ),
+        tool_path(
+            name = "clang",
+            path = "clang_wrappers/{os}/clang",
```

Comment:
I don't think this (or clang++) exist in the wrapper scripts.  I believe bazel always invokes the compiler with the tool named `gcc` (I haven't found where this happens yet).  The wrapper scripts allow the environment to dispatch to a different compiler as needed (e.g. clang, msvc, etc).